frontend/junit.xml

```diff
@@ -1,415 +1,592 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuites name="jest tests" tests="306" failures="0" time="48.146">
-  <testsuite name="FiltersActions general" errors="0" failures="0" skipped="0" timestamp="2021-03-16T13:44:10" time="3.471" tests="8">
-    <testcase classname="FiltersActions general should call DELETE_FILTER action with correct payload" name="FiltersActions general should call DELETE_FILTER action with correct payload" time="0.005">
+<testsuites name="jest tests" tests="310" failures="6" time="17.532">
```

Comment:
What's the deal with this thing ?

Reply:
accidentally added. Will remove it. Thanks for spotting this.